Added danger hint when using windows paths (#742)
ansys/pyacp: doc/source/intro.rst

See :class:`.FeFormat` for a list of supported FE formats. Check out the
:ref:`input_file_for_pyacp` section to see how input files can be created.

.. danger::

>     When working on Windows, be careful of possible **escape sequences** in paths that could result in errors.

Reviewer: Hm, I wonder if this is understandable if a user doesn't know what escape sequences are..

Maybe something like the below would be clearer?
"When working on Windows, be careful of *backslashes* in file paths: These need to be either escaped (written as double backslash `\\`), or you can use a *raw string literal* ..." 

@nshum4, do you have any input on this?

Reviewer 2: I agree that we should state backslashes instead of escape sequence. We should use plainest terms possible. 

@greschd I like your draft! Do the double backslashes in your draft have the same meaning as the "quote the backslashes" phrase from the original?

Reviewer: > Do the double backslashes in your draft have the same meaning as the "quote the backslashes" phrase from the original?

Yes, the backslashes are quoted.. with another backslash 😄

Author: Thanks for the suggestions!
I have modified the text to include the proper links to the documentation. I think it would be best to keep the proper naming for the problem that one might encounter, since in many cases a single `\` would not cause anything.
What do you think of this new version?

Reviewer 2: I wouldn't link outside our doc, unless necessary. What do you think about this revised version?

Danger: When working on Windows, be careful of backslashes that may result in errors. To avoid issues, quote the backslashes using an escape sequence (‘’\\’’) or use a raw string by prefixing the string with **r** (example: ``model = acp.import_model(r”path\to\your\model.acph5”)``).

Reviewer: I think it's good to link to the Python documentation here: we want to keep this info box short, and the Python documentation provides more background info.

I think the revised version is better; I made another suggestion to maybe make it more explicit that the error is related to backslashes. I would also be ok with merging the current version, though.

Author: I applied the suggestion, I think the final version is very clear. I will close the pull request if everyone is fine with the final version with the links to the python documentation.

See :class:`.FeFormat` for a list of supported FE formats. Check out the
:ref:`input_file_for_pyacp` section to see how input files can be created.

.. danger::

>     When working on Windows, be careful of possible 
>     `escape sequences <https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences>`_ in paths that could result in errors.

Reviewer: ```suggestion
    When working on Windows, be careful of backslashes in paths: These may correspond to
    `escape sequences <https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences>`_, resulting in errors.
```

Author: I like this version! Should we keep the links to the python documentation explaining the two concepts or should we remove them completely @nshum4 ?
